utils.py

```diff
@@ -72,3 +74,68 @@ def get_raw_data(fpath_set):
     assert len(raw.annotations) == 1200, err_msg
 
     return raw
+
+
+def get_behavioral_data(fpath_set):
+    """Read behavioral data to DataFrame.
+
+    Does the following:
+        - Reads in behavioral data as csv
+        - Checks if data has the expeceted length
+
+    Parameters
+    ----------
+    fpath_set : string
+        Absolute path to the csv file containing metadata
+
+    Returns
+    -------
+    metadata : pd.DataFrame
+        A pandas DataFrame containing metadata in columns and trials in rows.
+    """
+    #
+    metadata = read_csv(fpath_set)
+
+    # Set some known metadata
+    # Sanity check we have the expected number of events
+    err_msg = f"    >>> {len(metadata)} != the expected 1200"
+    assert len(metadata) == 1200, err_msg
+
+    return metadata
+
+
+def calculate_dPrime(behavior_dat):
+    """Calculate the sensitvity index for a given subject.
```

Comment:
I would not place this function in utils, but direcly in your `00` script --> utils should contain functions that get REUSED across different scripts: Do you expect that to happen with `calculate_dPrime` ? if yes, then fine by me :+1: if no, then it's easier for code readers to find this func where it is also used primarily

Reply:
sure.

Reply:
okay, let's have it in 00 then, and pull it back to utils if we do need it in the future